guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Implementation <s>in</s> <strong>using </strong>arcgis.learn <strong>module</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>meaning is conveyed, anyways.</p>